### DISCUSSION TITLE
960 geforce nvidia

### DISCUSSION THREAD

**Discussion by shqkn** [Lolliedieb/lolMiner-releases#262] [Q&A]
Hey, will a Nvidia Geforce GTX 960 be able to run 1.7 lolminer?

**Comment by yumyumfarted** [comment]
Yes, probably. Depending on the amount of VRAM your card has, certain coins might not be able to be mined. Here's an example: the DAG size for ETH is over 4GB, so you need a 6GB card or an 8GB. (There is a variant of the GTX 1060 with 5GB, but you can only get it in Asia).

I hope I was able to help, I know it probably wasn't as direct as you were hoping it to be.

**Reply by shqkn** [reply]
Ah ok I may just have to get a new graphics card or maybe an actual rig lol. Thanks for your help
